### ISSUE TITLE
Value of key in secrets should be base64 encoded

### ISSUE THREAD

**Issue by Shubham82** [kedacore/keda-docs#435]
In MSSQL scaler [example](https://keda.sh/docs/2.2/scalers/mssql/#example) in docs, The values for all keys in the data field in secrets should be base64-encoded. But  it mentioned as a string as shown below: 

```
apiVersion: v1
kind: Secret
metadata:
  name: mssql-secrets
type: Opaque
data:
  mssql-connection-string: "Server=example.database.windows.net;port=1433;Database=AdventureWorks;Persist Security Info=False;User ID=user1;Password=Password#1;Encrypt=True;TrustServerCertificate=False;"
``` 
we used stringData field if we want to mention string value. but for uniformity of the docs we should mentioned  base64 encoded value.

**Comment by Shubham82** [comment]
Hi @tomkerkhove @zroubalik  
could you please take a look!

**Comment by Shubham82** [comment]
Fixed it here #436
